pkgs/by-name/fc/fcitx5-rose-pine/package.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenvNoCC
+, fetchFromGitHub
+, lib
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "fcitx5-rose-pine";
+  version = "0-unstable-2024-03-01";
+  
+  src = fetchFromGitHub {
+    owner = "rose-pine";
+    repo = "fcitx5";
+    rev = "148de09929c2e2f948376bb23bc25d72006403bc";
+    hash = "0gywb8ykb2j3zyr20w1mc3f5xfw2aw4v9b821d55w36ja753k52a";
```

Comment:
Please set this to an empty string, and get the hash from the error when building (currently it's not an SRI hash)

Reply:
I don't even know what that means